### DISCUSSION TITLE
Clone APP or multiuser APP

### DISCUSSION THREAD

**Discussion by vantunes** [cgeo/cgeo#15139] [Ideas]
I would like to suggest that the possibility of cloning the APP for another user be created, similar to Whatsap.
I created an account for my son and I prefer to use C:Geo instead of another application, for both accounts.

**Comment by SammysHP** [comment]
Duplicate of #6621.

**Comment by eddiemuc** [comment]
@vantunes since you mention the possibility of cloning, maybe an app like https://appcloner.app/ is a solution for you? (I just googled this and haven't tried it out. However I would be interested whether c:geo would run in such a cloned mode)
